cmd/eksctl-anywhere/cmd/validations.go

```diff
@@ -33,3 +35,14 @@ func commonValidation(ctx context.Context, clusterConfigFile string) (*v1alpha1.
 	}
 	return clusterConfig, nil
 }
+
+func validateKubeVersion(kubeVersion string, source curatedpackages.BundleSource) error {
+	if source != curatedpackages.Registry {
+		return nil
+	}
+	versionSplit := strings.Split(kubeVersion, ".")
+	if len(versionSplit) < 2 {
```

Comment:
This allows `1.2.3.4`. We have a `semver` package that _should_ parse this appropriately, it might be better to use that logic.

Reply:
probably only support `== 2`. Maybe an e.g. on the error message `e.g.: 1.22`